easybuild/easyconfigs/h/hanythingondemand/hanythingondemand-3.0.0-cli.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonPackage'
+
+name = 'hanythingondemand'
+version = '3.0.0'
+versionsuffix = '-cli'
+
+homepage = 'https://github.com/hpcugent/hanythingondemand'
+description = """HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster.
+It uses the Torque resource manager to do node allocation."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+# minimal version for interactive use: just uses system Python (netaddr/netifaces/mpi4py not required)
+dependencies = [
+    ('pbs_python', '4.6.0'),
+    ('vsc-base', '2.4.2'),
+]
+
+use_easy_install = True
+zipped_egg = True
+
+# use 'hod' rather than 'hanythingondemand' in module name
+modaltsoftname = 'hod'
+
+options = {'modulename': 'hod'}
+
+modextravars = {
+    'HOD_BATCH_HOD_MODULE': 'hanythingondemand/3.0.0-beta-intel-2015b-Python-2.7.10',
```

Comment:
ah, and yes, hmm, the module name is indeed hardcoded, but others may be using a different naming scheme here, point taken...

I'll comment this out then, we can modify as needed when actually installing it (and HOD will work without having these set, as long as you use `--workdir` and `--hod-module`, and if not, it'll tell you).